test/JDBC/expected/catalogs_dbo_sys_schema-upgrade-vu-verify.out

```diff
@@ -0,0 +1,6872 @@
+~~END~~
+
+select object_name(id) from .tempdb..syscolumns where object_name(id) like 'tb_myschematest_%' order by 1
+go
+~~START~~
+varchar
+tb_myschematest_tempdb
+~~END~~
+
+select object_schema_name(id, db_id('myschematest_mydb')), object_name(id) from myschematest_mydb.dbo.syscolumns where object_name(id) like 'tb_myschematest_%' order by 1,2
+go
+~~START~~
+varchar#!#varchar
+dbo#!#tb_myschematest_mydb
+myschema#!#tb_myschematest_mydb
+~~END~~
+
+select object_schema_name(id, db_id('myschematest_mydb')), object_name(id) from .myschematest_mydb.dbo.syscolumns where object_name(id) like 'tb_myschematest_%' order by 1,2
+go
+~~START~~
+varchar#!#varchar
+dbo#!#tb_myschematest_mydb
+myschema#!#tb_myschematest_mydb
+~~END~~
+
+select object_schema_name(id, db_id('myschematest_mydb')), object_name(id) from myschematest_mydb.sys.syscolumns where object_name(id) like 'tb_myschematest_%' order by 1,2
+go
+~~START~~
+varchar#!#varchar
+dbo#!#tb_myschematest_mydb
+myschema#!#tb_myschematest_mydb
+~~END~~
+
+select object_schema_name(id, db_id('myschematest_mydb')), object_name(id) from .myschematest_mydb.sys.syscolumns where object_name(id) like 'tb_myschematest_%' order by 1,2
+go
+~~START~~
+varchar#!#varchar
+dbo#!#tb_myschematest_mydb
+myschema#!#tb_myschematest_mydb
+~~END~~
+
+select object_schema_name(id, db_id('myschematest_mydb')), object_name(id) from myschematest_mydb..syscolumns where object_name(id) like 'tb_myschematest_%' order by 1,2
+go
+~~START~~
+varchar#!#varchar
+dbo#!#tb_myschematest_mydb
+myschema#!#tb_myschematest_mydb
+~~END~~
+
+select object_schema_name(id, db_id('myschematest_mydb')), object_name(id) from .myschematest_mydb..syscolumns where object_name(id) like 'tb_myschematest_%' order by 1,2
+go
+~~START~~
+varchar#!#varchar
+dbo#!#tb_myschematest_mydb
+myschema#!#tb_myschematest_mydb
+~~END~~
+
+select cast(0 as smallint) from sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from .sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from ..sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from ...sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from dbo.sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from .dbo.sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from ..dbo.sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from master.dbo.sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from .master.dbo.sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
+select cast(0 as smallint) from master.sys.sysforeignkeys where 0 = 1 order by 1
+go
+~~START~~
+smallint
+~~END~~
+
```

Comment:
Why are we returning 0 rows? Was there no way to add deterministic test?

Reply:
It appears that other test cases are leaving schema elements behind. So while this test case is cleaning things up, it appears that in github, rows show up which originate from other tests.